business/checkers/virtual_services/route_checker.go

```diff
@@ -10,7 +10,9 @@ import (
 	"github.com/kiali/kiali/util/intutil"
 )
 
-type RouteChecker struct{ kubernetes.IstioObject }
```

Comment:
Interesting, didn't know this was possible (even though it seems not advisable :+1:)

Reply:
lol yeah you can do some interhitence in go...